container/apptabs_internal_test.go

```diff
@@ -47,3 +47,28 @@ func Test_tabButtonRenderer_DeleteAdd(t *testing.T) {
 	tabs.SelectTab(item2)
 	assert.Equal(t, pos, indicator.Position())
 }
+
+func Test_tabButtonRenderer_EmptyDeleteAdd(t *testing.T) {
+	item1 := &TabItem{Text: "Test", Content: widget.NewLabel("Content")}
+	tabs := NewAppTabs()
+
+	//cause tab button was build with enough space, without enough space, btns will not created.
```

Comment:
This doesn't read very clearly.
```suggestion
	// ensure enough space for buttons to be created.
```